authentication/handler_test.go

```diff
@@ -553,6 +553,31 @@ var _ = Describe("Handler", func() {
 		}`))
 	})
 
+	It("Allows access token authorization", func() {
+		// Prepare the next handler, which should not be called:
+		next := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			Expect(true).To(BeFalse())
```

Comment:
This test will fail now, as the handler is continuing on to `next`

Reply:
Fixing up. I was having some issues running the tests locally (passing?)